src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java

```diff
@@ -0,0 +1,88 @@
+package bogdanSandu.tests.Tema7;
+
+import bogdanSandu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class CURAHealthcare {
+    static ChromeDriver driver = null;
+
+    @BeforeMethod
```

Comment:
Can you please move @BeforeMethod and @AfterMethod to a separated abstract TestBase class and your test classes to extend that TessBase class? You have an example in the demo for chapter 7 from Timea. Thanks!